src/Client/Resources/keen-io-3_0.php

```diff
@@ -1235,7 +1216,6 @@
                     'description' => 'A Timeframe specifies the events to use for analysis based on a window of time. '
                                     . ' If no timeframe is specified, all events will be counted.',
                     'type'        => array('string', 'array'),
```

Comment:
Did you test to make sure it still works with either a single string, or an array of 1+ strings?

Reply:
Same comment for `group_by` where applicable, of course.

Reply:
Yeah, I tested all possible combinations and they work as long as they're actually valid timeframes. That includes a relative timeframe, which would be a string, as well as an "array", which is more like a dictionary, containing an absolute timeframe. More than one relative timeframe returns an error.

Reply:
`group_by` also works as expected with a single string or an array of strings.